### DISCUSSION TITLE
Opening the user login page is blocked

### DISCUSSION THREAD

**Discussion by shengwen-xue** [keycloak/keycloak#9493] [Q&A]
When I open the user login page, the page prompts me "We are sorry...  An internal server error has occurred”,
The following is the error screenshot and error log.
Hope it helps me, best wishes.

![image](https://user-images.githubusercontent.com/57387085/148895562-82ce79db-ead8-4ae0-8bfb-d7ab46f3bef4.png)
![image](https://user-images.githubusercontent.com/57387085/148895584-830d1dd0-27c2-4ba8-a763-4174237b6425.png)

14:51:18,465 ERROR [org.keycloak.services.error.KeycloakErrorHandler] (default task-1) Uncaught server error: org.keycloak.models.ModelException: Client account has browser flow override, but this flow does not exist

**Comment by stianst** [comment]
Exception is fairly self-explanatory here. Your client is using a custom flow that doesn't exist. Either remove the flow override from the client, or create the custom flow it is using.

**Reply by shengwen-xue** [reply]
I checked all the client side configs and didn't find any config that sets the browser to override the stream, but thank you very much for your answer.

![image](https://user-images.githubusercontent.com/57387085/148903823-e696751c-6991-446c-a5b0-f756d6fa6492.png)

**Reply by stianst** [reply]
Strange, wonder if it's somehow set it to an empty string then?

**Reply by shengwen-xue** [reply]
The version I'm using is 10.0.1, and after creating a new client, the default browser overlay stream is an empty one.
The following is the default configuration of a new test client and browser coverage flow I created.

![image](https://user-images.githubusercontent.com/57387085/149049197-fa092c28-1494-460e-bc46-3dd3f238b680.png)
![image](https://user-images.githubusercontent.com/57387085/149049217-1484df73-0824-4fb2-b6ca-ea3f2aa39001.png)

**Reply by shengwen-xue** [reply]
So I don't think the problem is here, but the error message in the log is indeed using a non-existent browser overlay stream.

**Comment by shengwen-xue** [comment]
Why is there no problem when a link is opened in a new tab in Chrome, but an error is reported when the iframe is accessed to a certain platform, why? But in firefox there is no problem at all.
I would like to know if it is browser compatible or needs to set parameters in keycloak.

Many thanks. 

![image](https://user-images.githubusercontent.com/57387085/155084257-d26f00ee-86ee-49f2-b1c0-3a306de7f147.png)

![image](https://user-images.githubusercontent.com/57387085/155084317-0198ecac-8ba9-4bc6-9327-33966bf790c6.png)

**Comment by lfdguaracy** [comment]
Did you ever find a solution for this?
